libkbfs/disk_block_cache.go

```diff
@@ -87,8 +88,6 @@ type DiskBlockCacheLocal struct {
 	closer func()
 }
 
-var _ DiskBlockCache = (*DiskBlockCacheLocal)(nil)
```

Comment:
Correct.  Since `DiskBlockCache` has so many methods that specify a cache type, it doesn't make sense for the local cache, which is already one of a specific type, to take that parameter.  And it's not used anywhere the generic interface is expected, so this makes everything cleaner.